distributionviewer/core/templates/distributionviewer/login.html

```diff
@@ -7,6 +7,14 @@
     <meta name="google-signin-client_id" content="{{ google_clientid }}">
     <title>Firefox Distribution Viewer</title>
     <link rel="stylesheet" type="text/css" href="{% static 'css/bundle.css' %}">
+    {% if user.is_authenticated %}
```

Comment:
If you wanted to go one step further you could do `if user.is_authenticated and not user.is_staff`. I think that's more clear of the intention also.